src/github-url.js

```diff
@@ -0,0 +1,131 @@
+const parseGithubUrl = require('parse-github-url');
+const fetch = require('node-fetch');
+
+const githubAPI = 'http://api.github.com';
+
+
+const isValidGithubUrl = (ghUrl) => ghUrl.protocol === 'https:'
```

Comment:
We're starting to get a bunch of "URL" specific things we want to do, @Cadesh has a PR up for one, @smilegodly has another, @KarlaChen has a third.  Later on, let's think about how we can combine some of them into common modules.  For now, duplication is OK.

Reply:
Probably a good idea to document that this is not a URL string, but a parsed URL Object you expect.

Reply:
I thought about waiting until @Cadesh's PR gets merged so I can use his tools to validate URLs, but I don't know when it's going to happen so I decided to duplicate. I'll change that once #74 gets merged.

Reply:
Sounds good, just add a comment above this re: url object vs string.